openedx/core/djangoapps/config_model_utils/models.py

```diff
@@ -316,30 +379,218 @@ def cache_key_name(cls, site, org, org_course, course_key):  # pylint: disable=a
 
         return super(StackedConfigurationModel, cls).cache_key_name(site_id, org, org_course, course_key)
 
-    @classmethod
-    def _org_from_org_course(cls, org_course):
-        return org_course.partition('+')[0]
+    def clean(self):
```

Comment:
method name does not seem to reflect the code

Reply:
@taniwha Could you expand on that a bit? I'm not sure I understand what the issue is.